docs/bgp.md

```diff
@@ -0,0 +1,161 @@
+
+	> calicoctl bgp peer add 192.20.30.40 as 64567
+
+To remove a global BGP peer that was configured with IP address 192.20.30.40 
+(the AS Number is not required when deleting) run the following command:
+
+	> calicoctl bgp peer remove 192.20.30.40
+
+To view the current list of global peers run the following command:
+
+	> calicoctl bgp peer show
+	+----------------------+--------+
+	| Global IPv4 BGP Peer | AS Num |
+	+----------------------+--------+
+	| 192.20.30.40         | 64567  |
+	+----------------------+--------+
+	+----------------------+--------+
+	| Global IPv6 BGP Peer | AS Num |
+	+----------------------+--------+
+	| aa:bb::ff            | 64514  |
+	+----------------------+--------+
+
+
+### Configuring a node specific BGP peer
+
+If your network topology requires specific peerings for each Calico node, you 
+can use the "calicoctl node bgp peer" commands to set up the peers specific to 
+a Calico nodes.  We refer to these as node specific peers.
+
+These commands need to be run on the specific node.
+
+For example, to add a BGP peer at IP address aa:bb::ff with AS Number 64514, 
+peering with Calico node "Node1", from the "Node1" command line run the
+following command:
+
+	Node1> calicoctl node bgp peer add aa:bb::ff as 64514
+
+To remove a BGP peer that was configured with IP address aa:bb::ff (the AS 
+Number is not required), peering with Calico node "Node1", from the "Node1" 
+command line run the following command:
+
+	Node1> calicoctl bgp peer remove aa:bb::ff
+
+To view the current list of node specific peers that are configured to peer
+with Calico node "Node1", from the "Node1" command line run the following
+command:
+
+	Node1> calicoctl node bgp peer show
+	No IPv4 BGP Peers defined for this node.
+
+	+-----------------------------+--------+
+	| Node specific IPv6 BGP Peer | AS Num |
+	+-----------------------------+--------+
+	| aa:bb::ff                   | 64514  |
+	+-----------------------------+--------+
+
+
+### Checking the status of the BGP peers
+
+To display the status of all BGP peerings for a specific node, use the 
+"calicoctl status" command.  This displays the status of all BIRD protocols 
+running on that node, and in particular will display the BGP status for each of 
+the peers - this includes the peers that are automatically configured as part
+of the full node-to-node mesh and the explicitly configured global peers and
+node specific peers.
+
+For example, to check the status of the peerings on Calico node "Node1", from 
+the "Node1" command line run the following command:
+
+	Node1> calicoctl status
+	calico-node container is running. Status: Up 43 seconds
+
+	IPv4 Bird (BGP) status
+	BIRD 1.5.0 ready.
+	+---------+----------+---------------+-------+----------+---------+
+	|   Name  | Protocol |   Peer type   | State |  Since   |   Info  |
+	+---------+----------+---------------+-------+----------+---------+
+	| kernel1 |  Kernel  |               |   up  | 21:59:07 |         |
```

Comment:
I wonder if there's any value in including the kernel, device and Direct protocol lines in this output. WDYT?